apps/examples/testcase/ta_tc/systemio/utc/utc_gpio.c

```diff
@@ -176,26 +170,12 @@ static void utc_gpio_set_drive_mode_p_IOTBUS_GPIO_DRIVE_FLOAT(void)
 	TC_SUCCESS_RESULT();
 }
 
-static void utc_gpio_set_drive_mode_p_IOTBUS_GPIO_DRIVE_PUSHPULL(void)
-{
-	TC_ASSERT_EQ("iotbus_gpio_set_drive_mode", iotbus_gpio_set_drive_mode(gpio, IOTBUS_GPIO_DRIVE_PUSHPULL), IOTBUS_ERROR_NONE);
-	TC_SUCCESS_RESULT();
-}
-
 static void utc_gpio_set_drive_mode_n(void)
 {
 	TC_ASSERT_EQ("iotbus_gpio_set_drive_mode", iotbus_gpio_set_drive_mode(NULL, -1), IOTBUS_ERROR_INVALID_PARAMETER);
 	TC_SUCCESS_RESULT();
 }
 
-static void utc_gpio_get_drive_mode_p(void)
```

Comment:
I think it's wrong way to delete this tc.
why don't you put the another drive mode instead of 'PUSHPULL'